elements/lisk-p2p/src/peer/base.ts

```diff
@@ -102,6 +103,7 @@ export interface PeerConfig {
 	readonly wsMaxPayload?: number;
 	readonly maxPeerInfoSize: number;
 	readonly maxPeerDiscoveryResponseLength: number;
+	readonly discoverySeedPeer?: boolean;
```

Comment:
I think you can use internal state and use peerKind to know if its a seed peer or not

Reply:
To avoid saving this flag and cause unwanted side effects later, I suggest that better to keep outside from internalState.